server/pulp/plugins/util/publish_step.py

```diff
@@ -902,7 +902,11 @@ def process_main(self, item=None):
 
         try:
             os.rename(self.source_dir, timestamp_master_dir)
-            selinux.restorecon(timestamp_master_dir.encode('utf-8'), recursive=True)
+            try:
+                selinux.restorecon(timestamp_master_dir.encode('utf-8'), recursive=True)
+            except OSError as e:
+                if e.errno != errno.EPERM:
+                    raise
```

Comment:
I like this better than the more crypic selinux_getenforcemode check in the previous commit.